addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,132 @@
+# iCloud Binding
+
+This binding uses the Apple iCloud service for providing device based information such as battery details or location.
+An Apple account is required. Two factor authentication is supported.
```

Comment:
Nice introduction! I'd suggest to add a link behind "Apple iCloud". We should always expect the user to not know the product.

How about https://www.apple.com/icloud/